src/io/utilities/functionSignatures.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I'm not sure I understand correctly this functionality. From the documentation of `tabComplete` I understand that it will generate the signature of the function automatically. Is this correct?
If so, I am wondering if it would be possible to put all the `json` files in a common directory. This is a little bit annoying to have all these `json` file in the `src` subfolders.